src/bundle/Resources/public/js/scripts/admin.location.visibility.js

```diff
@@ -1,12 +1,36 @@
-(function (global, doc) {
-    const updateVisibility = event => {
-        event.target.closest('.ez-checkbox-icon').classList.toggle('is-checked');
+(function(global, doc) {
+    const SELECTOR_VISIBILITY_CHECKBOXES = '#ez-tab-location-view-locations .ez-checkbox-icon__checkbox';
+    const fetchInit = {
+        credentials: 'same-origin',
+    };
+    const visibilityCheckboxes = [...doc.querySelectorAll(SELECTOR_VISIBILITY_CHECKBOXES)];
 
-        doc.querySelector('#location_update_visibility_data_location').value = event.target.value;
-        doc.querySelector('#location_update_visibility_data_hidden').checked = !event.target.checked;
+    const onVisibilityUpdated = (event) => {
+        const { target } = event;
+        const { checked: isHidden } = target;
```

Comment:
Do you want set a default value if none exists?

Reply:
Should I? I think that there is no default value in this case.